bulk-update/readme.md

```diff
@@ -25,17 +25,41 @@ Here is an example of a configuration object setup:
 config = {
   "siteUrl": "https://main--bacom--adobecom.hlx.page/", // The hlx site url
   "reporter": new ConsoleReporter(), // The logging type
-  "outputPath": "output", // The output directory for the docx files
+  "outputDir": "output", // The output directory for the docx files
   "mdDir": "markdown", // The directory for storing the fetched markdown.
   "cacheTimeMs": 0 // Disables the cache.
 }
 ```
 
+## Bulk Update
+
+The `bulk-update.js` file is responsible for loading data from various sources and performing a bulk update operation. It can be called using `npm run bulk-update <folder> <list>` from the root of the project.
+
+Examples are under the `migration-examples` folder.
+
 ## Migration Function
 
-The `migration.js` file sets up the configuration object and performs the actual migration.
+In order to create a migration, create a migration file within the a new migration folder with the name `migration.js`. This file is responsible for setting up the configuration object and performing the actual migration.
```

Comment:
nit: `In order to create a migration, create a migration file` -> In order to create a migration, add a migration file`

Reply:
Thanks